src/app/qgsdxfexportdialog.cpp

```diff
@@ -397,6 +397,81 @@ void QgsVectorLayerAndAttributeModel::applyVisibility( QSet<QString> &visibleLay
   }
 }
 
+void QgsVectorLayerAndAttributeModel::applyLayersOutputAttribute( const QStringList &encodedLayersOutputAttribute )
+{
+  QMap<QString, int> layersOutputAttribute;
```

Comment:
I think it'd be cleaner to save these as custom properties on each layer.

Reply:
@nyalldawson , done.